rocks-db/src/clients/asset_streaming_client.rs

```diff
@@ -254,7 +254,7 @@ async fn get_complete_asset_details(
             cli_tree_key: leaf.cli_tree_key,
             cli_node_idx: leaf.cli_node_idx,
         }),
-        asset_leaf: asset_leaf.map(|leaf| {
+        asset_leaf: asset_leaf.clone().map(|leaf| {
```

Comment:
do we really need to clone the whole thing? [u8; 32] is definitely `Copy`, so if `Flags` needs cloning then clone it separately pls